### PR TITLE
Don't fail if search endpoint is unavailable

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb \ @Eilon

File: src/NuGetPackageVerifier/Rules/PackageOwnershipRule.cs [hunk 1]
```diff
@@ -73,6 +74,22 @@ public IEnumerable<PackageVerifierIssue> Validate(FileInfo nupkgFile, IPackageMe
...
         }
 
+        private async Task<PackageSearchResult> GetPackageSearchResultAsync(IPackageVerifierLogger logger, string url)
+        {
+            using (var httpResponse = await _httpClient.GetAsync(url))
+            {
+                if (!httpResponse.IsSuccessStatusCode)
+                {
+                    logger.LogWarning($"{NuGetV3Endpoint} request failed. Response code {httpResponse.StatusCode}.");
+                    // The service might be unavailable. Return null here to fallback to screen scrapping the gallery.
```
Comment (Eilon) [inline src/NuGetPackageVerifier/Rules/PackageOwnershipRule.cs]: `scraping` 😄

**Comment by Eilon** [comment]
:shipit:

**Comment by ajaybhargavb** [comment]
:shipit:

**Pull request merged**